Source/WebKit/Shared/cf/CoreIPCCFDictionary.mm

```diff
@@ -55,6 +55,17 @@
     }];
 }
 
+// Helper function to check if an object is a valid property list object type
```

Comment:
I don't think this comment is necessary. This information is already conveyed by the function name.